controller/volume_controller.go

```diff
@@ -2281,7 +2281,7 @@ func (c *VolumeController) getReplicaCountForAutoBalanceZone(v *longhorn.Volume,
 		unusedZone[node.Status.Zone] = append(unusedZone[node.Status.Zone], nodeName)
 	}
 	if len(unusedZone) == 0 {
-		log.Info("Balanced, all ready zones are used by this volume")
+		log.Debugf("Balanced, all ready zones are used by this volume")
```

Comment:
How about the below log?

```go
log.Infof("Found %v zone available for auto-balance duplicates in %v", adjustCount, zoneExtraRs)
```

Reply:
This message is important scheduling decision message, need to keep in `Info`

Reply:
This message is important scheduling decision message, need to keep in `Info`